sycl/tools/sycl-xocc/bin/sycl-xocc

```diff
@@ -73,6 +73,8 @@ else
   # but a future TODO might be to allow this to be optionally specified as an
   # ENV variable
   XOCC_LIB_SPIR="$SDX_BIN_PATH_DIR/../lnx64/lib/libspir64-39-hls.bc"
+  # It says 3.9, but trust us this is Clang/LLVM 7..
```

Comment:
The problem is that if some day it is no longer version 7, we can no longer trust this comment...
So let's try to mitigate this with something like:
```bash
# It says 3.9, but this is Clang/LLVM 7 according to some urban legend...
```